packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -26,7 +26,7 @@
   ],
   "dependencies": {
     "@salesforce/core": "^3.31.8",
-    "@salesforce/eslint-config-lwc": "3.4.0",
+    "@salesforce/eslint-config-lwc": "3.3.3",
```

Comment:
This change was resulting in the error around a module `@babel/preset-typescript` missing.  We should probably open an issue over in that repo to verify it calls out if that module needs to get installed or what.

Reply:
It doesn't look like the LWC LSP even uses eslint-config-lwc https://github.com/search?q=repo%3Aforcedotcom%2Flightning-language-server+eslint-config-lwc&type=code

It does use "babel-types" and "babel-core": https://github.com/search?q=repo%3Aforcedotcom%2Flightning-language-server+babel&type=code

Reply:
this is a good callout. I'm not ever sure why this is a dependency in our repo. It might be tied to the whole eslint extension dep we have.  I'll create a ticket to check it out